docs/api/session.md

```diff
@@ -1511,7 +1511,7 @@ session is persisted on disk.  For in memory sessions this returns `null`.
 * `options` Object (optional)
   * `dataTypes` String[] (optional) - The types of data to clear. By default, this will clear all types of data.
     * `backgroundFetch` - Background Fetch
-    * `cache` - Cache
+    * `cache` - Cache (incl. cache storage)
```

Comment:
@nikwen looking at https://source.chromium.org/chromium/chromium/src/+/main:content/browser/browsing_data/browsing_data_remover_impl.cc;l=454;drc=5108636c70c0b08fdbeb57de2640a22e138f6685#:~:text=453-,454,-455, this also clears the shader cache.  Given that `ses.clearStorageData([options])` calls these `cachestorage` and `shadercache`, we should do the same here):
```suggestion
    * `cache` - Cache (includes `cachestorage` and `shadercache`)
```

Reply:
Good find, thanks!